### DISCUSSION TITLE
The remote server configuration are valid

### DISCUSSION THREAD

**Discussion by alwijein** [saran-surya/email_auth#69] [Q&A] [answered]
Help me sir, I don't understand why i have error like this, i have already following your tutorial until finish but still got error 

this my heroku link : https://email-auth-nodes.herokuapp.com

![image](https://user-images.githubusercontent.com/15154094/175801665-eba18b13-af25-4c3b-ae5e-a5e6ca351a0d.png)

thank later.

**Comment by saran-surya** [comment, marked as answer]
Hi @alwijein , As of latest information from google. they have removed less secure apps access to gmail, so i am looking into alternative solutions to work with, Will surely update the documentation once there is a valid solution.

**Reply by alwijein** [reply]
oke thank you sir. i hope you get valid solution for your package.

**Comment by saran-surya** [comment]
### Kindly use test server as of now.
Remote server patches are on the way, expect in 7 days

This should give you a good lead.
https://github.com/saran-surya/email_auth#update--google-turning-off-less-secure-apps--switching-to-oauth
